packages/zosjobs/src/ModifyJobs.ts

```diff
@@ -0,0 +1,108 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { AbstractSession, ImperativeExpect, Logger, Headers, ImperativeError } from "@zowe/imperative";
+import { JobsConstants } from "./JobsConstants";
+import { ZosmfRestClient } from "@zowe/core-for-zowe-sdk";
+import { IModifyJobParms } from "./doc/input/IModifyJobParms";
+import { IModifyJobOptions } from "./doc/input/IModifyJobOptions";
+import { IModifyJob } from "./doc/input/IModifyJob";
+import { IJobFeedback } from "./doc/response/IJobFeedback";
+
+
+/**
+ * Class to handle modify of jobclass and holdStatus information
+ * @export
+ * @class ModifyJobs
+ */
+export class ModifyJobs {
+
+    /**
+     * Modify a job
+     * @static
+     * @param {AbstractSession} session - z/OSMF connection info
+     * @param {IModifyJobParms} parms - parms object (see IModifyJobParms interface for details)
+     * @param {IModifyJobOptions} options - options object (see IModifyJobOptions interface for details)
+     * @returns {Promise<undefined|IJobFeedback>} - promise of undefined, or IJobFeedback object returned by API if modifyVersion is 2.0
+     * @memberof ModifyJobs
+     */
+    public static async modifyJob(session: AbstractSession,
+        parms: { jobname: string, jobid: string },
```

Comment:
Should this be using IModifyJobParms and IModifyJobOptions instead so they inherit any future changes?